src/zoom.css

```diff
@@ -0,0 +1,10 @@
+.zoom-enter, .zoom-leave-to{
+	transform: scale(2);  
```

Comment:
I think this animation is very similar to the `shrink` and `scale-in` animations already added.